### PR TITLE
Set DCN for higher pytorch version

### PR DESCRIPTION
**Pull request by Owen-Liuyuxuan**
DCN borrowed from https://github.com/lbin/DCNv2/tree/pytorch_1.7.

### CODE REVIEW

**Comment by zhangyp15** [comment]
Thanks for your contributions. Have you tried this version of DCN for torch 1.4?

Or maybe we should set two branches for different versions of pytorch.

**Comment by Owen-Liuyuxuan** [comment]
I managed to build my fork at CUDA111/torch1.8, CUDA110/torch1.7, and also CUDA 102/torch1.5. Though there are different warnings, it works.

I do not maintain a torch 1.4 environment so perhaps you could help verify that?

I have also trained on the validation set with CUDA111/torch 1.8 and basically verified your results on the validation set.

**Comment by zhangyp15** [comment]
Thanks, I will test the higher environments and create a new branch.
